src/cli/commands/compile.rs

```diff
@@ -0,0 +1,29 @@
+use std::{fs::File, io::Write, path::PathBuf, process::Command};
+
+const JSON_FILE_EXTENTION: &str = "json";
+
+pub fn compile(program: &PathBuf) -> PathBuf {
+	// Use cairo-compile binary in order to compile the .cairo file
+	let compiled_output = Command::new("cairo-compile")
+		.args([&program])
+		.output()
+		.expect("cairo-compile command failed to start");
+
+	// Retrieve only the file name to create a clean compiled file name.
+	let filename = program.file_stem().unwrap();
+
+	// Build path to save the  compiled file
+	let dir = dirs::cache_dir().unwrap();
```

Comment:
Rather than `unwrap` use `expect` with an error message saying that this operating system is not supported by the dirs crate.